hack/aks/Makefile

```diff
@@ -207,16 +236,18 @@ windows-cniv1-up: rg-up overlay-net-up ## Bring up a Windows CNIv1 cluster
 
 	$(AZCLI) aks nodepool add --resource-group $(GROUP) --cluster-name $(CLUSTER) \
```

Comment:
might also need K8S_VER?

Reply:
It doesnt. It takes version from linux nodes.